src/Promotions/Campaigns/CampaignsWelcomeBanner.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Give\Promotions\Campaigns;
+
+namespace Give\Promotions\Campaigns;
+
+use Give\Vendors\StellarWP\AdminNotices\AdminNotice;
+use Give\Vendors\StellarWP\AdminNotices\AdminNotices;
+use Give\Vendors\StellarWP\AdminNotices\DataTransferObjects\NoticeElementProperties;
+
+/**
+ * @unreleased
+ */
+class CampaignsWelcomeBanner
+{
+    /**
+     * @var string
+     */
+    public $id = 'givewp-campaigns-welcome-banner-2025';
+
+    /**
+     * @unreleased
+     */
+    public function render()
+    {
+        AdminNotices::show($this->id, [$this, 'renderCallback'])
+            ->custom()
+            ->location('below_header')
+            ->dismissible()
+            ->enqueueStylesheet(GIVE_PLUGIN_URL . 'build/campaignWelcomeBannerCss.css', [], '1.0.0')
+            ->enqueueScript(GIVE_PLUGIN_URL . 'build/campaignWelcomeBannerJs.js', [], '1.0.0')
+            ->on('plugins.php');
+    }
+
+    /**
+     * @unreleased
+     */
+    public function renderCallback(AdminNotice $notice, NoticeElementProperties $elements): string
+    {
+        $backgroundUrl = GIVE_PLUGIN_URL . 'assets/src/images/admin/promotions/campaigns/welcome-banner-background.svg';
+        $badgeIconUrl = GIVE_PLUGIN_URL . 'assets/src/images/admin/promotions/campaigns/star-badge-icon.svg';
+        $heartIconUrl = GIVE_PLUGIN_URL . 'assets/src/images/admin/promotions/campaigns/heart-icon.svg';
+        $exitIconUrl = GIVE_PLUGIN_URL . 'assets/src/images/admin/promotions/campaigns/dismiss-icon.svg';
+        $campaignsPageUrl = admin_url('admin.php?page=give-campaigns');
+
+        return "
```

Comment:
Hey man, would it be too much to ask that you create a separate file for this template and then load it with our `View` helper class? I mean, this works, obviously, but it will be much cleaner if we have these things separate.

Reply:
Sounds good to me! Thanks! 
Resolved: 9d0821e05dc3e97e04a5872c85f65c259fad5a4f